Open Judge System/OJS.Services/Data/Participants/IParticipantsDataService.cs

```diff
@@ -10,31 +10,35 @@
 
     public interface IParticipantsDataService : IService
     {
-        void Add(Participant participant);
-
-        void Update(Participant participant);
-
-        void Update(
-            IQueryable<Participant> participantsQuery,
-            Expression<Func<Participant, Participant>> updateExpression);
-
-        void Delete(IEnumerable<Participant> participants);
+        Participant GetByContestByUserAndByIsOfficial(int contestId, string userId, bool isOfficial);
 
         Participant GetWithContestByContestByUserAndIsOfficial(int contestId, string userId, bool isOfficial);
 
-        IQueryable<Participant> GetByIdQuery(int participantId);
+        IQueryable<Participant> GetByIdQuery(int id);
 
         IQueryable<Participant> GetAll();
 
         IQueryable<Participant> GetAllByContestAndIsOfficial(int contestId, bool isOfficial);
 
-        bool AnyByContestIdUserIdAndIsOfficial(int contestId, string userId, bool isOfficial);
-
-        bool IsOfficialById(int participantId);
-
         IQueryable<Participant> GetAllOfficialInOnlineContestByContestAndContestStartTimeRange(
```

Comment:
Get all official Participants in online contest, since by convention we do not use the name of the model for which the service is. The context is clear when the method is being called like this `participantsData.GetAllOfficialInOnlineContest`